src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  width: #{$calendar-size};
+  padding: 10px;
+
+  @for $index from 28 through 31 {
+    &--month-length-#{$index} {
+      .calendar__day:nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: #{$cell-size};
+    height: #{$cell-size};
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid black;
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
```

Comment:
For colors and transition time it is better to create variables
Please fix all cases

Reply:
I've fixed it. Thank You :)